applications/sandbox/src/App.js

```diff
@@ -162,6 +162,8 @@ function App() {
     }
   };
 
+  console.log('map layers', layers);
```

Comment:
Remove it

Reply:
I'm going to merge, because I remove it in the next PR